### PR TITLE
Added currencySymbol in detailed response

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added new property currencySymbol in Detailed response when DestinationRate object is requested.

### CODE REVIEW

File: library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php [hunk 1]
```diff
@@ -34,6 +34,21 @@ public static function getPropertyMap(string $context = '', string $role = null)
             ];
         }
 
+        if ($context === self::CONTEXT_DETAILED) {
+            return [
+                'cost' => 'cost',
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php:39]: This should change tests

File: library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php [hunk 2]
```diff
@@ -34,6 +34,21 @@ public static function getPropertyMap(string $context = '', string $role = null)
...
+                'destinationRateGroupId' => 'destinationRateGroup',
+                'destinationId' => 'destination',
+                'tpRateId' => 'tpRate',
+                'tpDestinationRateId' => 'tpDestinationRate',
+                'currencySymbol' => 'currencySymbol',
+            ];
+        }
+
         return parent::getPropertyMap(...func_get_args());
     }
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php:53]: Show currencySymbol in new portal

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php [hunk 3]
```diff
@@ -34,6 +34,21 @@ public static function getPropertyMap(string $context = '', string $role = null)
...
+            return [
+                'cost' => 'cost',
+                'connectFee' => 'connectFee',
+                'rateIncrement' => 'rateIncrement',
+                'groupIntervalStart' => 'groupIntervalStart',
+                'id' => 'id',
+                'destinationRateGroupId' => 'destinationRateGroup',
+                'destinationId' => 'destination',
+                'tpRateId' => 'tpRate',
+                'tpDestinationRateId' => 'tpDestinationRate',
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php:47]: This fields are missing in getDestinationRate.feature

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php (hunk 3 above)
Comment (ironBatista) [reply library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php:47]: These fields are not being used, but it was there before this new feature, what should I do?

- [ ] Delete it.
- [ ] Set them in the Assembler.

**Review by mmadariaga — APPROVED** [review]

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**